convince-my-boss.html

```diff
@@ -46,9 +46,11 @@
     <!-- <div id="preloader">
     <div id="load"></div>
   </div> -->
-    <nav class="navbar navbar-custom navbar-fixed-top" style="-webkit-box-shadow: 0 1px 30px rgba(0, 0, 0, .1);">
+    <nav class="navbar navbar-custom navbar-fixed-top top-nav-collapse" style="-webkit-box-shadow: 0 1px 30px rgba(0, 0, 0, .1);">
```

Comment:
I am not sure if this class is required as it isn't present in other pages. Can you remove this and check ?

Reply:
No, this class displays PyCon logo. I have tried to remove it PyCon logo disappears.